### DISCUSSION TITLE
Device UI

### DISCUSSION THREAD

**Discussion by Anghellos** [merdok/homebridge-miot#282] [Ideas]
Hello,

I would like to know if there is any way to display values in some device properties as the image bellow instead of separate buttons.
Example on vacuum instead of having 5 different buttons for the speed have one with all the options inside
![IMG_0065](https://user-images.githubusercontent.com/35992448/172017467-1fc3f0b3-4366-4ccb-a3f4-11f1e5f1afed.jpg)

**Comment by merdok** [comment]
Hi, it would be possible to use that, but the issue is that you cannot change the amount of buttons here and you cannot rename the buttons so it would basically look like on your screenshot all the time without any info what does what.
